src/app/utilities/utilities.js

```diff
@@ -0,0 +1,11 @@
+export const isUserLoggedIn = () => {
+  const userEmail = localStorage.getItem("userEmail");
+  const userRole = localStorage.getItem("userRole");
+
+  if(userEmail && userRole) {
```

Comment:
What if this condition is false?

Reply:
Then the function will return an undefined object, stating that there is no user currently remembered by the browser.
Is it better practice to return false if the condition is not met?

Reply:
You should consider the naming convention in React, so when we use the name `is...`, the expected output should be `true` or `false` instead of the object.
One more thing, if there are only 2 options to return, we can eliminate `else`, please let me know if this is still vague.

Reply:
I get it now, thanks. My current solution is to have the function renamed to getUserFromBrowser, as it was my intented use of the function.